README.md

```diff
@@ -38,10 +38,9 @@ Create a file at any place you want. Usually it would be called bin/check though
 You can now configure your scripts like that:
 
 ```rb
-# Include all ruby files of renuo-bin-check
-Dir["<path-to-renuo-bin-check>/lib/renuo/bin-check/*.rb"].each { |file| require file }
-
 # Initialize bin-check
+require 'renuo-bin-check'
```

Comment:
suppi, aber mer muss glaub no di linie ändere wi mer sGem sött is Gemfile inetue 😊